index.js

```diff
@@ -20,8 +20,60 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    let firstIteration = Recipe.create(data[0])
+     firstIteration
+        .then((res)=>{
+              console.log("data was sent",res)
+    })
+        .catch(()=>{
+              console.log("failure")
+    })
+  
+  
+  let secondIteration = Recipe.insertMany(data)
+    secondIteration
+        .then((res)=>{
+              res.filter((res)=>{ console.log(res.title)})
```

Comment:
filter is used when you want to create a new array. If you just need to run a code for each element, then it is better to use the forEach method.